lotto/client/js/utils/templates.js

```diff
@@ -0,0 +1,13 @@
+export const winningLottoTemplate = data => {
+  return "<div class='lotto'>" + data.winningNumbers
+    .map(number => `<div class="lotto-number winning">${number}</div>`)
+    .concat(`<div class="lotto-number winning bonus-number">${data.bonus}</div>`)
+    .join("") + "</div>";
```

Comment:
통일성 있게 모두 [Template literals](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Template_literals)를 활용하는 것이 더 좋겠네요.
`${data.winningNumbers.map(...)}` 부분도 함수로 분리한다면 가독성이 더 좋아질 것 같습니다.

```suggestion
  return `
  <div class='lotto'>
   ${data.winningNumbers.map(number => `<div class="lotto-number winning">${number}</div>`).join("")}
    <div class="lotto-number winning bonus-number">${data.bonus}</div>
  </div>
`;
```

Reply:
```
export const winningLottoTemplate = data => {
  return `
    <div class='lotto'>
        ${makeWinningLotto(data.winningNumbers)} 
        <div class="lotto-number winning bonus-number">${data.bonus}</div>
    </div>
    `;
}
```

```
function makeWinningLotto(winningNumbers) {
  return winningNumbers.map(number => `<div class="lotto-number winning">${number}</div>`).join("");
}
```
로 수정했습니다.